gapir/client/session.go

```
@@ -327,6 +328,8 @@ func (s *session) newADB(ctx context.Context, d adb.Device, abi *device.ABI, lau
 	}
 
 	log.I(ctx, "Launching GAPIR...")
+	// Configure GAPIR to be traceable for replay profiling
+	perfetto_android.SetupRenderStagesEnvironment(ctx, d, apk.Name, true, abi, []string{})
```

Don't ignore the error or the cleanup here.